packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -636,10 +662,49 @@ const updateConfigFromModel = (inputModel: Model) => {
 	knobs.value.parametersMetadata = inputModel.metadata?.parameters ?? {};
 };
 
-const createConfiguration = async () => {
+const runSanityCheck = () => {
+	const errors: string[] = [];
+	const modelToCheck = modelConfiguration.value?.configuration as Model;
+	if (!modelToCheck) {
+		errors.push('no model defined in configuration');
+		return errors;
+	}
+
+	modelToCheck.semantics?.ode?.parameters?.forEach((p) => {
+		const val = p.value || 0;
+		const max = p.distribution?.parameters.maximum;
+		const min = p.distribution?.parameters.minimum;
+		if (val > max) {
+			errors.push(`${p.id} value ${p.value} > distribution max of ${max}`);
+		}
+		if (val < min) {
+			errors.push(`${p.id} value ${p.value} < distribution min of ${min}`);
+		}
+
+		// Arbitrary 0.01 here, try to ensure interval is significant w.r.t value
+		const interval = Math.abs(max - min);
+		if (val !== 0 && Math.abs(interval / val) < 0.01) {
+			errors.push(`${p.id} distribution range [${min}, ${max}] may be too small`);
+		}
+	});
+	return errors;
+};
+
+const createConfiguration = async (force: boolean = false) => {
 	if (!model.value) return;
 
 	const state = cloneDeep(props.node.state);
+
+	sanityCheckErrors.value = [];
+	if (force !== true) {
+		const errors = runSanityCheck();
+		if (errors.length > 0) {
+			console.log(errors);
```

Comment:
Just to make it more digestible
```suggestion
      console.log("Model configuration sanity check errors: ");
			console.table(errors);
```